content/en/docs/zero-code/java/spring-boot/starter/getting-started.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Getting Started
+description: Getting Started of the OpenTelemetry starter
+weight: 20
+# prettier-ignore
+cSpell:ignore: autoconfigurations springboot
+---
+
+## OpenTelemetry Spring Boot starter
+
+### Compatibility
+
+The OpenTelemetry Spring Boot starter works with Spring Boot 2.0 and 3.0, and
```

Comment:
need to update that with version range based on outcome of http://localhost:1313/docs/zero-code/java/spring-boot/starter/getting-started/

Reply:
It does not seem related to the reorganization of the Spring Boot content. You could perhaps elaborate this point in a Github issue?